lib/msal-core/src/AuthenticationParameters.ts

```diff
@@ -24,6 +24,7 @@ export type AuthenticationParameters = {
     account?: Account;
     sid?: string;
     loginHint?: string;
+    force?: boolean;
```

Comment:
I think I would prefer `skipCache` over `force`, is more clear what it will be used for.

Reply:
https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/976b5e26798d04716e82ea215b12e3374afa7d40/src/Microsoft.Identity.Client/ApiConfig/AcquireTokenForClientParameterBuilder.cs

How about this for consistency with .NET 

`forceRefresh`